osu.Game/Screens/Play/HUD/LegacyComboSplash.cs

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Skinning;
+using osuTK;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    public class LegacyComboSplash : Container<Container<Sprite>>, ISkinnableDrawable
+    {
+        [SettingSource("Side, where bursts will appear")]
+        public Bindable<Side> BurstsSide { get; } = new Bindable<Side>(Side.Random);
+
+        public Bindable<int> Current { get; } = new BindableInt { MinValue = 0 };
```

Comment:
This bindable shouldn't be public (or if you really want it to be, it should be externally immutable by exposing as `IBindable<int>`).